tests/utils/object_store/object_store_settings.py

```diff
@@ -26,11 +27,20 @@
 except ImportError:
     _BOTO3_AVAILABLE = False
 
+try:
+    import paramiko
+    _PARAMIKO_AVAILABLE = True
+    del paramiko
+except ImportError:
+    _PARAMIKO_AVAILABLE = False
+
 _object_store_marks = {
     LibcloudObjectStore: [pytest.mark.skipif(not _LIBCLOUD_AVAILABLE, reason='Missing dependency')],
     LibcloudObjectStoreHparams: [pytest.mark.skipif(not _LIBCLOUD_AVAILABLE, reason='Missing dependency')],
     S3ObjectStore: [pytest.mark.skipif(not _BOTO3_AVAILABLE, reason='Missing dependency')],
     S3ObjectStoreHparams: [pytest.mark.skipif(not _BOTO3_AVAILABLE, reason='Missing dependency')],
+    SFTPObjectStore: [pytest.mark.skipif(not _PARAMIKO_AVAILABLE, reason='Missing dependency')],
```

Comment:
I think the tests will require some additional setup to be able to mock paramiko.